tests/README.md

```diff
@@ -0,0 +1,16 @@
+# Acceptance Tests for Renkulab Images
```

Comment:
Should this just be added to the main README? The relative paths in the instructions are wrong if it stays in this directory

Reply:
I fixed the paths so that they are all as if the user has already navigated to the `test` directory.

I think the main readme is already long enough and all the test commands need the user to be in the `test` directory anyhow.